src/main/java/com/exactpro/th2/codec/csv/CsvCodec.java

```diff
@@ -155,6 +156,27 @@ public void handler(String consumerTag, RawMessageBatch message) {
         }
     }
 
+    private void fillCsvHeaderMessage(Message.Builder headerMsg, String[] strings) {
+        ListValue.Builder headerValues = ListValue.newBuilder();
+        for (String string : strings) {
+            headerValues.addValues(Value.newBuilder().setSimpleValue(string).build());
```

Comment:
`Value.newBuilder().setSimpleValue(string).build()` can be replaced with `ValueUtilsKt.toValue(string)`

Reply:
Thanks) There is an overload for an array of strings by the way. It returns the Value with the ListValue set. I will use it here but I will leave the current version in tests because we don't have tests for the common library and those utility methods...